src/test/java/org/reaktivity/specification/nukleus/oauth/streams/ProxyConnectIT.java

```diff
@@ -256,4 +256,17 @@ public void shouldForwardRequestWithoutBearerOnUnsecuredRoute() throws Exception
         k3po.finish();
     }
 
+    @Test
+    @Specification({
+        "${streams}/reauthorize.inflight.request.same.privileges.update.expiration/connect/client",
+        "${streams}/reauthorize.inflight.request.same.privileges.update.expiration/connect/server"
```

Comment:
Suggestions for test scenario names.

`earlier.expiring.authorization.does.not.reauthorize.inflight.request`
`later.expiring.authorization.reauthorizes.inflight.request`
`later.expiring.authorization.with.more.roles.reauthorizes.inflight.request`
`later.expiring.authorization.with.fewer.roles.does.not.reauthorize.inflight.request`
`later.expiring.authorization.with.different.affinity.does.not.reauthorize.inflight.request`
`later.expiring.authorization.with.different.subject.does.not.reauthorize.inflight.request`